android/src/main/java/com/pagopa/ioreactnativecrypto/IoReactNativeCryptoModule.kt

```diff
@@ -396,6 +398,18 @@ class IoReactNativeCryptoModule(reactContext: ReactApplicationContext) :
     }
   }
 
+  @ReactMethod
+  fun unpackBerEncodedASN1(asn1Signature: String, coordinateOctetLength: Int, promise: Promise) {
+    try {
+      val decodedBytes = Base64.decode(asn1Signature, DEFAULT)
+      val transcoded = transcodeSignatureToConcat(decodedBytes, coordinateOctetLength)
+      val signature = Base64.encodeToString(transcoded, DEFAULT)
+      promise.resolve(signature)
+    } catch (ex: Exception) {
+      promise.reject(ex)
```

Comment:
What do you think to integrate with our standard way to reject by extending our `ModuleException` and provide a specific exception type also exporting it to RN?